src/applications/yellow-ribbon/components/SearchResult/index.jsx

```diff
@@ -0,0 +1,84 @@
+// Node modules.
+import React from 'react';
+import PropTypes from 'prop-types';
+import { invoke } from 'lodash';
+// Relative imports.
+import { capitalize } from '../../helpers';
+
+const SearchResult = ({ school }) => (
+  <div className="medium-screen:vads-l-col vads-l-col vads-u-background-color--gray-light-alt vads-u-margin-bottom--2 vads-u-padding-x--3 vads-u-padding-y--2">
+    {/* School Name */}
+    <h3 className="vads-u-margin--0">{capitalize(school?.name)}</h3>
+
+    {/* School Location */}
+    <p className="vads-u-margin-bottom--1 vads-u-margin-top--0">
+      {capitalize(school?.city)}, {school?.state}
```

Comment:
This is the kind of thing that makes me wonder about the `?` operator. It's nice that it won't cause a runtime error, but also we may expect that `city` is always present. And if not, we won't to render the comma afterwards. 🤷‍♂

Reply:
Ya I think this whole label should just be derived as a separate variable tbh... going to tweak this :)

Reply:
https://github.com/department-of-veterans-affairs/vets-website/pull/11551#discussion_r372523130 Did the same thing here! 🙂